src/sentry/search/events/datasets/discover.py

```diff
@@ -1020,6 +1024,18 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
                     ),
                     private=True,
                 ),
+                SnQLFunction(
+                    "salted_column_hash",
+                    # TODO: figure out how to make this take arbitrary number of arbirary type arguemnts
+                    required_args=[SnQLStringArg("salt"), ColumnArg("column")],
+                    snql_aggregate=lambda args, alias: Function(
+                        "farmFingerprint64",  # farmFingerprint64 aka farmHash64 is a newer, faster replacement for cityHash64
```

Comment:
I would just double check with SnS if this hash function is available in the clickhouse versions in prod.

Reply:
Good point, it's been added in [v20.11.2.1, 2020-11-11](https://clickhouse.com/docs/en/whats-new/changelog/2020#clickhouse-release-v201121-2020-11-11), I'll make sure that we use this or newer version.

Reply:
I haven't confirmed production version yet, the image we're using in dev uses **21.6.1.6734**